.env.example

```diff
@@ -5,3 +5,6 @@ DISCORD_TOKEN="❤"
 DISCORD_PUBLIC_KEY="❤"
 DISCORD_WEBHOOK_URL="❤"
 DISCORD_CHANNEL_ID="❤"
+
+CF_ACCOUNT_ID="🧡"
+CF_API_TOKEN="🧡"
```

Comment:
Not required by core LC-Dailies program.

Reply:
Done in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/ffd3a3e189f56a573b7c270947ba239ddbef61df>.